src/Util.py

```diff
@@ -4,47 +4,47 @@
 @author: r4stl1n
 '''
 
+
 def fileContentsToList(fileName):
-    
     lineList = []
-    
+
     try:
         fileParser = open(fileName, 'r')
-        
+
     except IOError:
-        print "[!] Could not open file %s " % fileName
-        
+        print("[!] Could not open file " + fileName)
+
     except:
-        print "[!] Could not access file %s" % fileName
-        
+        print("[!] Could not access file " + fileName)
+
     for line in fileParser.readlines():
         newLine = line.replace('\n', '')
         lineList.append(newLine)
-        
+
     return lineList
 
+
 def fileContentsToTuple(fileName):
-    
     tupleList = []
-    
+
     try:
         fileParser = open(fileName, 'r')
-        
+
     except IOError:
-        print "[!] Could not open file %s " % fileName
-        
+        print("[!] Could not open file " + fileName)
+
     except:
-        print "[!] Could not access file %s" % fileName
-        
+        print("[!] Could not access file " + fileName)
+
     for line in fileParser.readlines():
         newLine = line.replace('\n', '')
-        newTuple = (newLine[:line.find(':')],newLine[line.find(':')+1:])
+        newTuple = (newLine[:line.find(':')], newLine[line.find(':') + 1:])
```

Comment:
Maybe change this to :
newTuple = newLine.split(':')

Since this is creating not the correct solution cuz "find function will return -1 when it doesn't find the specified pattern".